### DISCUSSION TITLE
[ALL]그라운드 룰 상세화 논의

### DISCUSSION THREAD

**Discussion by MoonJeWoong** [woowacourse-teams/2023-haru-study#11] [아이디어]
# 그라운드 룰
## 9. 과정은 즐거우나 결과는 철저하게

**Comment by MoonJeWoong** [comment]
## 1. 사용자 없는 개발은 없다.

**Reply by woosung1223** [reply]
우리가 만드는 서비스는 사용자를 위해 존재합니다. 따라서 모든 의사결정에서 사용자의 입장을 배제해서는 안됩니다!

**Reply by jaehee329** [reply]
모든 개발의 목적은 사용자의 편익을 최우선으로 합니다. 기술에 대한 궁금증, 개발 과정의 편리함 등은 후순위입니다.

**Reply by MoonJeWoong** [reply]
우리의 서비스는 사용자에게 긍정적인 경험을 선사하기 위해 기획되었음을 잊지 말아야 합니다.

**Reply by woo-jk** [reply]
우리의 서비스는 사용자의 입장에 기반하여 생각합니다. 또한, 개발자가 고생해야 사용자가 편리하다는 마인드로 개발합니다.

**Comment by MoonJeWoong** [comment]
## 2. 팀원의 고민은 곧 나의 고민이다.

**Reply by jaehee329** [reply]
우리 팀은 같은 목표를 향해 나아가며, 팀원의 고민을 나의 것으로 생각하고 언제나 적극 도와야 합니다.

**Reply by MoonJeWoong** [reply]
팀원의 고민은 개인이 아니라 우리 팀이 함께 해결해야 할 문제라고 인식합니다.

**Comment by MoonJeWoong** [comment]
## 3. 검토 없는 결정은 없다.

**Reply by woosung1223** [reply]
팀원 몰래 독자적으로 결정하는 일은 없어야 합니다. 의견이 생기면 팀원과 나누고 함께 결정해요.

**Reply by MoonJeWoong** [reply]
팀의 방향을 개인이 결정할 수 없습니다. 반드시 팀이 검토하고 팀이 결정합니다.

**Reply by jaehee329** [reply]
의견과 변경 사항은 언제나 팀원들의 적극적인 검토를 거쳐야 합니다. 협업이 아닌 분업이 되지 않도록 합니다.

**Comment by MoonJeWoong** [comment]
## 4. 같이 결정했다면 같이 책임진다.

**Reply by nlom0218** [reply]
하루스터디는 우리 모두가 함께 결정하여 만듭니다. 누군가를 탓하지 말고 우리의 의사 결정을 되돌아보며 함께 책임집니다.

**Reply by jaehee329** [reply]
- 함께 내린 결정에 대해선 모두가 균등한 책임을 가집니다. 발의한 사람에게 책임을 돌리지 않습니다.
- 다수의 판단이 잘못되었다 느껴진다면 따라가지 말고 끝까지 설득해야 합니다.

**Comment by MoonJeWoong** [comment]
## 5. 질문이 없다면 되돌아봐라.

**Reply by woosung1223** [reply]
질문을 두려워하지 않아야 합니다. 질문은 팀의 성장을 위한 지름길입니다.

**Reply by nlom0218** [reply]
질문이 없다면 내가 생각하고 있는 것이 맞는지 의문을 가집니다.

**Reply by MoonJeWoong** [reply]
질문이 없다고 생각이 드는 순간 정말 모든 것을 이해한 것인지 다시 한 번 확인합니다.

**Comment by MoonJeWoong** [comment]
## 6. 칭찬에 엄지를 아끼지 마라.

**Reply by nlom0218** [reply]
잘 한 것에 대한 칭찬은 우리를 더욱 행복하게 만듭니다. 명확한 몸짓으로 서로를 칭찬하여 팀의 분위기를 올립니다.

**Reply by MoonJeWoong** [reply]
비난은 쉽게 드러나지만, 칭찬을 확실하게 표현하지 않으면 상대방이 알 수 없습니다. 
칭찬을 속으로만 생각하지 말고 반드시 표현합니다.

**Reply by woosung1223** [reply]
돈은 아낄수록 모이지만, 칭찬은 아껴도 남는게 없습니다. 보다 적극적으로 팀원들을 칭찬해 사기를 복돋아야 합니다.

**Comment by MoonJeWoong** [comment]
## 7. 근거 없는 의견은 불평이다.

**Reply by jaehee329** [reply]
마음에 들지 않는 사항이 있다면 충분한 이유와 함께 팀원들을 설득해야 합니다. 근거 없이 불평만을 늘어놓는 것은 팀의 사기를 저하시킵니다.

**Reply by MoonJeWoong** [reply]
근거 없는 의견은 쉽게 튀어나오고 논의의 본질을 흐리기 쉽습니다. 항상 의견 제시는 근거를 기반으로 합니다.

**Reply by nlom0218** [reply]
의견을 제시하기 전 머리속으로 다시 정리합니다. 나의 의견과 근거가 명확하고 설득 가능성이 있다면 말합니다. 그렇지 않는 것은 훌러가는 불평입니다.

**Comment by MoonJeWoong** [comment]
## 8. 비교 대상은 우리의 어제뿐이다.

**Reply by nlom0218** [reply]
비교를 통한 성장은 없습니다. 하루스터디의 목표는 어제보다 더 나은 오늘입니다.

**Reply by woosung1223** [reply]
우리는 타인과 비교하기보다는, 회고를 통해 과거 팀의 모습과 비교합니다.

**Reply by woo-jk** [reply]
다른 팀의 진행도가 빠르다고 해서 조급해하지 않고 우리는 우리만의 길을 나아가는데 집중합니다.

**Comment by MoonJeWoong** [comment]
## 9. 과정은 즐거우나 결과는 철저하게

**Reply by woosung1223** [reply]
`개발자가 힘들어야 사용자가 즐겁다` 라는 말이 있습니다. 하지만 우리는 `개발자도 즐겁고 사용자도 즐거운` 서비스를 만드는 것에 도전합니다.

**Reply by jaehee329** [reply]
팀으로 활동하는 전 과정이 즐겁도록 노력해야 합니다. 동시에 모든 결과물의 높은 완성도를 지향합니다.

**Reply by MoonJeWoong** [reply]
하루스터디는 자유롭고 즐거운 분위기를 추구합니다. 그러나 결과를 내야 할 때는 누구보다 이성적으로 판단하고 행동합니다.
